example_plugins/baz/src/registerEndpoints.ts

```diff
@@ -60,7 +60,8 @@ export function registerEndpoints(
       // the validated object may not have this now
       // This is where BazService's underlying elasticsearch service
       // should be bound to the right cluster
-      const bazService = new BazService(req.headers, config, elasticsearch);
+      const cluster: Cluster = await elasticsearch.getScopedCluster('admin', req.headers);
```

Comment:
You don't need `: Cluster` here, as `getScopedCluster` already returns a `Cluster`.